resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -520,9 +521,12 @@ general.execution.failed=failed
 general.execution.running=running...
 general.execution.success=completed successfully
 general.file_not_found=File not found: "{0}"
+general.help=Help
 general.in_progress_button=In progress
 general.message=Message
 general.name.label=Name:
+general.notification.action.hide_once=Dismiss
+general.notification.action.hide_forever=Don't show again
```

Comment:
Possible duplicate of `notice.suppress`, which is pretty much the same message

Reply:
Removed notice.suppress